xdsl/dialects/accfg.py

```diff
@@ -72,117 +105,58 @@ def __init__(self, accelerator: str | StringAttr):
         return super().__init__([accelerator])
 
 
-class AcceleratorSymbolOpTrait(SymbolOpInterface):
+@irdl_op_definition
+class LaunchOp(IRDLOperation):
     """
-    Symbol op trait to define multiple accelerator names.
+    Launch an accelerator. This acts as a barrier for CSR values,
+    meaning CSRs can be safely modified after a launch op without
+    interfering with the Accelerator.
     """
 
-    def get_sym_attr_name(self, op: Operation) -> StringAttr | None:
-        assert isinstance(op, AcceleratorOp)
-        return StringAttr(op.name_prop.string_value())
-
+    name = "accfg.launch"
 
-@irdl_op_definition
-class AcceleratorOp(IRDLOperation):
```

Comment:
This op was moved further down the document, sadly the diff isn't very clear here. Can re-arrange if needed